distribution/sender/routes/db-sync-route.xml

```diff
@@ -17,6 +17,8 @@
                 #{body.setDateCreated(new java.util.Date())}
                 #{body.setMessageUuid(T(java.util.UUID).randomUUID().toString())}
                 #{body.setRequestUuid(getProperty('event').requestUuid)}
+                #{body.setEventDate(body.getDateCreated())}
```

Comment:
I don't these changes work, please read the ticket descriptions carefully, you should copy this date from the debezium event

Reply:
I also don't see where you have updated the route test